packages/ui/src/working-groups/modals/CreateOpening/types.tsx

```diff
@@ -0,0 +1,109 @@
+import { EventRecord } from '@polkadot/types/interfaces/system'
+import BN from 'bn.js'
+import * as Yup from 'yup'
+
+import { QuestionValueProps } from '@/common/components/EditableInputList/EditableInputList'
+import { ModalWithDataCall } from '@/common/providers/modal/types'
+import { BNSchema, minContext, minMixed } from '@/common/utils/validation'
+import { GroupIdName } from '@/working-groups/types'
+
+export interface OpeningModalData {
+  group: GroupIdName
+}
+
+export type CreateOpeningModalCall = ModalWithDataCall<'CreateOpening', OpeningModalData>
+
+// TODO research runtime constraints
+export interface OpeningConditions {
+  group: GroupIdName
+  minStake: BN
+  hiringTarget: number
```

Comment:
```suggestion
    minUnstakingPeriodLimit?: BN,
    minimumApplicationStake?: BN,
```
I'm not sure about the types though